docs/src/components/Theming/TokenList.tsx

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import { useTheme } from '@aws-amplify/ui-react';
+import {
+  SpaceBlock,
+  BorderWidthBlock,
+  FontFamilyBlock,
+  FontSizeBlock,
+  LineHeightBlock,
+  FontWeightBlock,
+  ColorBlock,
+} from './TokenBlocks';
+
+export function createTokenList(tokens) {
+  // Creates a flattened array out of the token object passed to createTokenList()
+  let tokenList = [];
+  function iterateGroup(group) {
+    Object.values(group).forEach((value) => {
+      if (typeof value === 'object' && value.hasOwnProperty('name')) {
+        tokenList.push({
+          ...value,
+        });
+      } else {
+        iterateGroup(value);
+      }
+    });
+  }
+  if (tokens.hasOwnProperty('name')) {
+    tokenList.push(tokens);
+  } else {
+    iterateGroup(tokens);
+  }
+  return tokenList;
+}
```

Comment:
Any feedback on a better way to do this? We need a nice flat array to map over for TokenList, and `tokens` can be different levels of nested-ness (e.g. tokens.space vs tokens.colors.neutral)

Reply:
I think this is good